sdk/src/main/scala/com/horizen/account/storage/AccountStateMetadataStorageView.scala

```diff
@@ -243,6 +245,8 @@ class AccountStateMetadataStorageView(storage: Storage) extends AccountStateMeta
   }
 
   def setCeased(): Unit = hasCeasedOpt = Some(true)
+  def setForgerListOpen(flag: Boolean): Unit = isForgersListOpenOpt = Some(flag)
```

Comment:
I don't see anyone using this method and I don't see that we store `isForgersListOpenOpt` flag.
I expect, that all the changes in this file can be reverted, since we keep all the data inside `ForgerStakeMsgProcessor`

Reply:
You are right, it was a refuse, done

Reply:
2 more places left in this file:
https://github.com/HorizenOfficial/Sidechains-SDK/pull/640/files#diff-666349b00787560e252c851b079d9ac69b5ee99d5516f9dbcf5c1063174b1094R36
https://github.com/HorizenOfficial/Sidechains-SDK/pull/640/files#diff-666349b00787560e252c851b079d9ac69b5ee99d5516f9dbcf5c1063174b1094R265